crates/locutus-dev/src/bin/local_node.rs

```diff
@@ -1,7 +1,14 @@
 use clap::Parser;
```

Comment:
we can rename the utility to just `locutus-dev` for now

Reply:
renamed